src/GovUk.Education.ExploreEducationStatistics.Common/Model/DataSetScreenerResponse.cs

```diff
@@ -1,22 +1,15 @@
 #nullable enable
-
-using System;
 using System.Collections.Generic;
 using System.Text.Json.Serialization;
 
-namespace GovUk.Education.ExploreEducationStatistics.Admin.Models;
+namespace GovUk.Education.ExploreEducationStatistics.Common.Model;
 
-public class DataScreenerResponse
+public class DataSetScreenerResponse
```

Comment:
I'm guessing that this could be a record, but not suggesting doing anything like that as part of this PR!